Automated Smoketest: :x:
OS: centos 7.2.1511 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 93acf24c138385bad461e0e4116d4ea69a58d38a
Build: success
ECL Watch:  Rebuilding Site
npm ERR! file /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/tmp/package.json
npm ERR! code EJSONPARSE
npm ERR! Failed to parse json
npm ERR! Unexpected token < in JSON at position 1480 while parsing near \...es/dojo: 1.9.40,
npm ERR! <<<<<<< HEAD
npm ERR!     @h...\
npm ERR! File: /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/tmp/package.json
npm ERR! Failed to parse package.json data.
npm ERR! package.json must be actual JSON, not just JavaScript.
npm ERR! 
npm ERR! Tell the package author to fix their package.json file. JSON.parse
npm ERR! A complete log of this run can be found in:
npm ERR!     /mnt/disk1/home/vamosax/.npm/_logs/2018-04-12T13_54_48_685Z-debug.log
CMake Warning at /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/cmake_install.cmake:55 (message):
  npm install failed see above and
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/build_install.txt
  for details.
Call Stack (most recent call first):
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/cmake_install.cmake:41 (include)
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/cmake_install.cmake:50 (include)
CMake Warning at /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/cmake_install.cmake:60 (message):
  npm run copy-res failed see above and
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/build_copy.txt for
  details.
Call Stack (most recent call first):
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/cmake_install.cmake:41 (include)
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/cmake_install.cmake:50 (include)
CMake Warning at /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/cmake_install.cmake:65 (message):
  npm run compile failed see above for details.
Call Stack (most recent call first):
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/cmake_install.cmake:41 (include)
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/cmake_install.cmake:50 (include)
CMake Warning at /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/cmake_install.cmake:69 (message):
  npm run bundle failed see above and
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/src/build_bundle.txt
  for details.
Call Stack (most recent call first):
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/esp/cmake_install.cmake:41 (include)
  /mnt/disk1/home/vamosax/smoketest/PR-10884/build/cmake_install.cmake:50 (include)
CPack: Create package

Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
Adding hpcc to group hpcc ...\x1b[32m [   OK    ]  \x1b[0m 
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |85 |85 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |759 |759 |0 |
|test (thor) |686 |686 |0 |
|test (roxie) |799 |799 |0 |


HPCC Stop: OK
HPCC Uninstall: OK


 ... ( comment is too long, 84 lines, 3847 bytes, truncated) ... 
[Build error:](https://gist.github.com/daa9ea1875f17c6dbbb50d3bd1412f9f)
Build log [Build log](https://gist.github.com/HPCCSmoketest/daa9ea1875f17c6dbbb50d3bd1412f9f#file-result-18-04-12-09-51-16-log)

Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   16 sec (00:00:16) |  186 sec (00:03:06) |   57 sec (00:00:57) |    6 sec (00:00:06) |   21 sec (00:00:21) | 1099 sec (00:18:19) |   16 sec (00:00:16) | 1401 sec (00:23:21) |

```

--- a/esp/src/src/ESPLogicalFile.ts
+++ b/esp/src/src/ESPLogicalFile.ts
@@ -239,7 +239,7 @@ var LogicalFile = declare([ESPUtil.Singleton], {    // jshint ignore:line
         var context = this;
         WsDfu.DFUInfo({
             request: {
-                FileName: this.Name,
+                Name: this.Name,
                 Cluster: this.Cluster,
                 UpdateDescription: true,
                 FileDesc: description

